foundry.toml

```diff
@@ -3,7 +3,7 @@ src = "src"
 out = "out"
 libs = ["lib"]
 solc = "0.8.17"
-
+viaIR = true
```

Comment:
What's this for?

Reply:
emits out `stack to deep error`
removed added it from the `toml` and explicitly added in ci test command.